### ISSUE TITLE
VID2015 and Center loss

### ISSUE THREAD

**Issue by pabloe4993** [foolwood/DCFNet#7]
Hi,

Thank you for great job.
 I have few questions:

1. In the previous version DCFNet trained on uav123, nus-pro and tc-128.
the current one is defined to VID2015, why?
2. Why you don't use the CenterLoss loss anymore?  Why in the first place the CenterLoss don't propagate(only forward but no backward)?

Thank you

**Comment by foolwood** [comment]
1. The recent trend is to use VID to train. 

- First, the amount of video in VID is very large（~1 million images, and more than 4,000 videos）. This number is far exceeding all the tracking datasets. 

- On the other side, there is a common belief that there is a risk of overfitting when training on the tracking datasets. VOT committee expressly prohibit training on this dataset([Learning from the tracking datasets (OTB, VOT, ALOV, NUSPRO) is prohibited.](http://www.votchallenge.net/vot2017/participation.html)).

2. CenterLoss inherited from [SiamFC](https://github.com/bertinetto/siamese-fc/blob/161c5dd7b2bee920aa5f6251660d8fc1ed99fa2c/training/experiment.m#L189) directly to visualize the convergence conditions. 
- But I find it's too slow and provides no additional information. So I decide to del these loss.
- CenterLoss is only used to visualize the convergence (Like Top1/5 in image classification). This loss is  non-differentiable( just think top1 loss. Only cross entropy can bp).

Thanks for your attention.
